elkm1_lib/keypads.py

```diff
@@ -61,6 +71,14 @@ def _kc_handler(self, keypad: int, key: int) -> None:
             name = ""
         self.elements[keypad].setattr("last_keypress", (name, key), True)
 
+    def _kf_handler(self, keypad: int, key: str, chime_mode: list[int]) -> None:
+        self.elements[keypad].last_function_key = None  # Force a change notification
+        try:
+            name = FunctionKeys(key).name
+        except ValueError:
+            name = ""
+        self.elements[keypad].setattr("last_function_key", (name, key), True)
```

Comment:
Should this be `(name, key.value)`?

Reply:
Sure, kc_handler passes the key thru, but it takes in an int for the key. kf_handler takes in an enum and passes that thru.  So it's your your call if it passes thru the enum or the value.